vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -143,33 +152,104 @@
             function onMouseLeave(e) {
                 $(this).removeClass("hova");
             }
+
+            function getSelectedSpace(blockId) {
+                $.ajax({
+                    url: "[(@{'/staff/module/'+${module.id}+'/slide/'+${slide.id}+'/space/'})]" + blockId,
+                    type: 'GET',
+                    success: function(result) {
+                    	if (result != null && result.space != null && result.space.name != null){
+                            currentSpaceName = result.space.name;
+                            $("#selectedSpace").text(currentSpaceName);
+                            $("#editSpaceBlock").text(result.title);
+                            currentSpaceId = result.space.id;
```

Comment:
put this line together with line 162

Reply:
and honestly, not sure if you even need extra variables here. You can just use `result.space.name` and `result.space.id` everywhere.